docs/guides/reusing_functions.md

```diff
@@ -51,9 +51,7 @@ classes will need:
 import numpy as np
 ```
 
-To add a setup cell in the editor, use the General menu and select "Add setup cell" (💠).
-
-Setup cells are guaranteed to run before all other cells.
+To add a setup cell in the editor, open the notebook menu and select "Add setup cell".
```

Comment:
I still don't think the options is super findable by itself, and the latter statement is true

Reply:
Yea requires some searching. The emoji I found confusing, we could find a way to get the lucide icon in here at some point but didn't want to set that up. Added the second sentence back but in parentheses, since it isn't the point we're trying to get across.